content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -46,14 +46,14 @@ cluster for mesh expansion, run the following commands on a machine with cluster
     $ cd -
     {{< /text >}}
 
-    *   If you installed Istio without Helm and Tiller, use `helm template` to update your configuration with the option and
-        reapply with `kubectl`:
+    *   If you installed Istio without Helm and Tiller, use `helm template` to update your configuration with the option and reapply with `kubectl`:
```

Comment:
I've previously tried adding the `global.meshExpansion.enabled=true` flag to a pre-existing Istio deployment and never was able to get that to work. What did work for me was to do a full, clean install of Istio with that flag.

Has anyone here verified that adding this flag to a pre-existing Istio deployment actually works?

Reply:
I lack the helm/tiller/upgrade experience to tell if these doc are inaccurate or we miss some prerequisite/info ourselves...